src/duplicatesRemoval.java

```diff
@@ -0,0 +1,39 @@
+import java.util.ArrayList;
+
+public class duplicatesRemoval {
+
+	
+	
+	
+	
+	
+public static void removeDuplicates(ArrayList<String> list) {
+		int dNum = 0;
+		int lSize = list.size();
+		for (int i = 0; i < list.size() - 1; i++) {
+			for (int j = i + 1; j < list.size(); j++) {
+				if (!list.get(j).equals(list.get(i))) continue;
+				dNum++; //assumes there is a duplicate term
+				list.remove(j);
+				j--;
+				lSize --;
+			}
+		}
```

Comment:
This works perfectly fine but would be slow. If you know what a big O is, the big O of this function you wrote is O(n^2). There would be a more efficient way to run this that uses an ArrayList to store all the checked ones. The O is O(n) for that function.
But watch out for the shift of index after you remove something, which you did a good job